src/pages/DetailsPage.js

```diff
@@ -83,14 +83,17 @@ class DetailsPage extends React.PureComponent {
 
     render() {
         let details = lodashGet(this.props.personalDetails, lodashGet(this.props.route.params, 'login'));
+        const login = lodashGet(this.props.route.params, 'login');
+
         if (!details) {
-            const login = lodashGet(this.props.route.params, 'login');
+            const avatar = ReportUtils.getAvatar(lodashGet(details, 'avatar', ''), login);
             details = {
                 login,
                 displayName: ReportUtils.getDisplayNameForParticipant(login),
-                avatar: ReportUtils.getDefaultAvatar(),
+                avatar,
             };
         }
```

Comment:
NAB - I meant I thought something more like this would be cleaner:

```js
        if (!details) {
            const login = lodashGet(this.props.route.params, 'login');
            details = {
                login,
                displayName: ReportUtils.getDisplayNameForParticipant(login),
                avatar: ReportUtils.getAvatar(lodashGet(details, 'avatar', ''), login),
            };
        }
```

Reply:
Since we don't use `login` anywhere except in this if condition, and it's easy to inline the `avatar: ReportUtils.getAvatar` part